src/beamlime/config/raw_detectors/bifrost.py

```diff
@@ -0,0 +1,279 @@
+import scipp as sc
+
+_res_scale = 8
+pixel_noise = sc.scalar(4.0, unit='mm')
+
+detectors_config = {
+    'detectors': {
+        'Channel (1, 1)': {
+            'detector_name': '123_channel_1_1_triplet',
+            'resolution': {'y': 30 * _res_scale, 'x': 20 * _res_scale},
+            'projection': 'xy_plane',
+            'pixel_noise': pixel_noise,
+        },
```

Comment:
Is there no need for the projection? I thought I need to use this to apply the transformations and offsets in the nexus file. The LogicalView doesn't run through that information right?